lib/src/gestures/interactive_flag.dart

```diff
@@ -25,6 +25,7 @@ abstract class InteractiveFlag {
       scrollWheelZoom |
       rotate;
 
+  /// Use this flag if you want to disable all gesture interactions.
```

Comment:
Avoid 'you', and avoid explaining WHY this should be used in the first sentence of a doc. Instead explain WHAT is does.

https://dart.dev/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs

```suggestion
  /// Disable all gesture interactions
```